### ISSUE TITLE
Reflection + clipping error

### ISSUE THREAD

**Issue by gesievi** [Archtica/Radii#79]
When having an active clipping plae with reflective materials:
The reflection stays in position when moving sideways as shown in the picture. 
Only observed with clipping plane active, when toggling the plane the reflection artifact disappears temporarily.   
[bug.pdf](https://github.com/Archtica/RADii/files/13717194/bug.pdf)

**Comment by Archtica** [comment]
Reflections are not dynamic, but only changing when changing the weather or time. This is a limitation of the engine because reflection comes from a cubemap that is updated once in a while only. It gives the impression of reflection, but they are never true reflections based on POV. I will look into it if something else is going on.

**Comment by gesievi** [comment]
Yes they are not dynamic, but they stay in place on the screen when moving during clipping. 
The brown coloured area is the artefact  from the windows to the right side of it. in the picture i moved left a bit and the window reflection stayed in position on my screen instead of disappearing.

**Comment by Archtica** [comment]
I will look into it. I'm not sure I can see what you mean in the image, but I'll do a live test.
